00 Intro/01 JSX/src/components/panel-component/PanelComponent.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+
+interface Props {
+  title: string;
+  children?: React.ReactNode | React.ReactNode[];
```

Comment:
Yes, I don't need a stateful `<PanelComponent />` because it's only a presentational frame.